### PR TITLE
feat: add shm size to start options

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingyang@tensorchord.ai>

* fix https://github.com/tensorchord/envd/issues/1429

### CODE REVIEW

**Comment by kemingy** [comment]
Blocked by https://github.com/tensorchord/envd-server/pull/176

File: pkg/app/run.go [hunk 1]
```diff
@@ -74,6 +74,11 @@ var CommandCreate = &cli.Command{
 			Usage: "Assign the host address for environment ssh acesss server listening",
 			Value: envd.Localhost,
 		},
+		&cli.IntFlag{
+			Name:  "shm-size",
+			Usage: "Configure the shared memory size (megabyte)",
+			Value: 0,
```
Comment (gaocegege) [inline pkg/app/run.go:80]: I think we should set a larger default value, WDYT?

Comment (kemingy) [reply pkg/app/run.go:80]: How much? It also consumes memory. The default resource limit is 2GB of memory. 
- https://github.com/tensorchord/envd-server/blob/f9e9d2689a22edda828be867dde626eb188c6824/pkg/runtime/kubernetes/environment_create.go#LL176C30-L176C30

Comment (kemingy) [reply pkg/app/run.go:80]: Be default, the container will have 64MB of shared memory.

Comment (gaocegege) [reply pkg/app/run.go:80]: I think for docker, we can keep 2GB memory. WDYT

Comment (kemingy) [reply pkg/app/run.go:80]: SGTM

**Review by gaocegege — APPROVED** [review]
/lgtm

**Pull request merged**